src/__tests__/redis-client.factory.ts

```diff
@@ -2,9 +2,16 @@ import type { RedisClientType } from 'redis';
 import { createClient } from 'redis';
 
 export async function redisClientFactory(): Promise<RedisClientType> {
-  const { REDIS_HOST = 'localhost', REDIS_PORT = 6379 } = process.env;
+  const {
+    REDIS_USER = undefined,
+    REDIS_PASS = undefined,
```

Comment:
These are `undefined` by default.

```suggestion
    REDIS_USER,
    REDIS_PASS,
```

Reply:
Done
https://github.com/safe-global/safe-client-gateway/pull/2222/commits/9e76bd1f48294ecc71a0e57f6d30cf1a18d746c0